code/modules/uplink/uplink_items.dm

```diff
@@ -1737,6 +1737,15 @@ datum/uplink_item/stealthy_tools/taeclowndo_shoes
 	item = /obj/item/clothing/glasses/thermal/syndi
 	cost = 3
 
+/datum/uplink_item/device_tools/syndi_auth
+	name = "Syndicate Authentication Device"
+	desc = "This device grants an individual temporary authentication to Syndicate networks, preventing them being marked as hostile by turrets and other technology."
+	item = /obj/item/faction_aligner_syndicate
+	cost = 1
+	surplus = 0
+	include_modes = list(/datum/game_mode/nuclear)
```

Comment:
The only way I could see this being even remotely useful is on dynamic, and in that case it would first need them to buy the tool assuming there will be traitors, find the traitors instead of shooting on sight and give it to them. This item is extremely situational and depends on an unlikely game mode combo and also requires the nukies to buy it beforehand anyway, so I don't really see this ever being purchased.

Reply:
I had a dynamic round this week as a nukie war op and several traitors bought syndie encryption keys so we were coordinating before we even arrived. We rescued 3 of them but only the malf AI was able to get on the shuttle alive. 

I intend to do some changes that allow dynamic to be added to normal rotation, this is in part to prep for that.